task/queue/config.go

```diff
@@ -0,0 +1,28 @@
+package queue
+
+import (
+	"runtime"
+	"time"
+)
+
+// Config configuration for task queue.
+type Config struct {
+	// NumWorkers  number goroutine instances.
+	NumWorkers int `yaml:"numWorkers" json:"numWorkers"`
```

Comment:
It's not number of workers, it's goroutinePoolSize . If you add here 10000 it doesn't mean you have 10000 workers handling in parallel goroutines from the pool

Reply:
Fixed